### PR TITLE
Add docker-compose support for local kinesis/dynamo dev nodes.

### PR DESCRIPTION
**Pull request by johngian**
* Add boto3 dependency.

### CODE REVIEW

**Comment by johngian** [comment]
Example queries with `boto3`:

```
In [8]: c = boto3.client('kinesis', endpoint_url='http://127.0.0.1:4568/')

In [9]: c.list_streams()
Out[9]: 
{u'HasMoreStreams': False,
 'ResponseMetadata': {'HTTPHeaders': {'connection': 'keep-alive',
   'content-length': '41',
   'content-type': 'application/x-amz-json-1.1',

   ....

  'RetryAttempts': 0},
 u'StreamNames': []}

In [10]: c = boto3.client('dynamodb', endpoint_url='http://127.0.0.1:4567/')

In [11]: c.list_tables()
Out[11]: 
{'ResponseMetadata': {'HTTPHeaders': {'connection': 'keep-alive',
   'content-length': '25',
   'content-type': 'application/x-amz-json-1.0',
   'date': 'Thu, 13 Apr 2017 22:26:13 GMT',

   ....

  'RetryAttempts': 0},
 u'TableNames': [u'Movies']}
```

**Comment by johngian** [comment]
@andrewkrug or @akatsoulas r?

File: dockerfiles/Dynalite [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM node:alpine
+EXPOSE 4567
+RUN apk add --update --no-cache python make g++
```
Comment (akatsoulas) [inline dockerfiles/Dynalite]: Are those deps needed for Dynalite?

Comment (johngian) [inline dockerfiles/Dynalite]: yes

**Pull request merged**